scripts/tests

```diff
@@ -360,6 +366,28 @@ def bench(force_qemu, verbose, cycles, cflags, arch_flags):
         [f"CYCLES={cycles}"],
     )
 
+    if output:
+        import json
+
+        with open(output, "w") as f:
+            v = []
+            for scheme in results:
+                schemeStr = str(scheme)
+                r = results[scheme]
+                d = {
+                    k: int(v)
+                    for k, v in (l.decode().split("=") for l in r.splitlines()[:3])
+                }
+                for primitive in ["keypair", "encaps", "decaps"]:
+                    v.append(
+                        {
+                            "name": f"{schemeStr} {primitive}",
+                            "unit": "cycles",
+                            "value": d[f"{primitive} cycles"],
```

Comment:
Nit, why append "cycles" if we've specified the unit before?

Reply:
It does not append 'cycles'. That's the key for looking it up from the output.

Reply:
ah, got it